analyzers/dataframe/ReconstructedParticle2Track.cc

```diff
@@ -11,6 +11,16 @@ ROOT::VecOps::RVec<float> getRP2TRK_D0(ROOT::VecOps::RVec<edm4hep::Reconstructed
   return result;
 }
 
+ROOT::VecOps::RVec<float> getRP2TRK_D0_var(ROOT::VecOps::RVec<edm4hep::ReconstructedParticleData> in,  ROOT::VecOps::RVec<edm4hep::TrackState> tracks) {
```

Comment:
var sounds like variable. what about using cov?

Reply:
"var" was for "variance", but I am fine with "cov". I have updated consistently the examples validation_tkParam.py and validation_tkParam_plots.x in the last commit.